lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -56,6 +58,10 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
       ..color = const Color(0x000000000)
       ..blendMode = BlendMode.dstIn;
 
+    rangeAnnotationPaint = Paint()
+      ..style = PaintingStyle.fill
+      ..color = const Color(0xf5eaf1f6);
```

Comment:
This line is not needed, because you set the color before you use the `rangeAnnotationPaint` every time and it is not used.

Reply:
6e4386d